components/test/index.js

```diff
@@ -0,0 +1,8 @@
+import '@testing-library/jest-dom';
+import { render } from '@testing-library/react';
+
+export function snapshot(element, name) {
+    test(name, () => {
+        expect(render(element).container).toMatchSnapshot();
+    });
+}
```

Comment:
The reusable function, to avoid repetitive calls to `test`, `expect`, `render` and `toMatchSnapshot`